pkg/sdk/system_functions.go

```diff
@@ -26,21 +27,24 @@ type systemFunctions struct {
 	client *Client
 }
 
-func (c *systemFunctions) GetTag(ctx context.Context, tagID ObjectIdentifier, objectID ObjectIdentifier, objectType ObjectType) (string, error) {
+func (c *systemFunctions) GetTag(ctx context.Context, tagID ObjectIdentifier, objectID ObjectIdentifier, objectType ObjectType) (*string, error) {
 	objectType, err := normalizeGetTagObjectType(objectType)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
 	s := &struct {
-		Tag string `db:"TAG"`
+		Tag sql.NullString `db:"TAG"`
 	}{}
 	sql := fmt.Sprintf(`SELECT SYSTEM$GET_TAG('%s', '%s', '%v') AS "TAG"`, tagID.FullyQualifiedName(), objectID.FullyQualifiedName(), objectType)
 	err = c.client.queryOne(ctx, s, sql)
 	if err != nil {
-		return "", err
+		return nil, err
+	}
+	if !s.Tag.Valid {
```

Comment:
I'm not sure if this method should be modified as much, I was modified to fit into implementation. I don't mind leaving return value as string and in this situation returning error that the value wasn't found or just returning empty string.

Reply:
Why not? Now we can detect if tag was empty, and there's a difference between empty and nil tags.

Reply:
Ok, then it makes sense.